src/qibolab/result.py

```diff
@@ -1,100 +1,91 @@
-from dataclasses import dataclass
 from functools import cached_property
 from typing import Optional
 
 import numpy as np
 import numpy.typing as npt
 
-ExecRes = np.dtype([("i", np.float64), ("q", np.float64)])
+iq_record_dtype = np.dtype([("i", np.float64), ("q", np.float64)])
```

Comment:
I would avoid calling it `dtype`, and even `record`: it is a `dtype` and a composite one, so it is already available information.
It is more or less like if you were calling `QubitClass` the `Qubit` class.